backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/tourplanning/api/IDeliveryDayGenerator.java

```diff
@@ -22,4 +22,6 @@
 	void setDateFrom(final LocalDate dateFrom);
 
 	void generate(String trxName);
+
+	void generateDeliveryDaysForTourVersion(String trxName, I_M_TourVersion tourVersion);
```

Comment:
pls call it `generateForTourVersion` 
...we allready guessed that i created delviery days ;-)

Reply:
Done!